cg/meta/upload/vogue.py

```diff
@@ -4,25 +4,41 @@
 
 from cg.apps.gt import GenotypeAPI
 from cg.apps.vogue import VogueAPI
+from cg.store.api.findbasicdata import FindBasicDataHandler
```

Comment:
```suggestion
from cg.store import Store
```
The store provides an API, I suggest using it instead of digging so deep in the implementation..